src/entities/movie/hooks/useGetTrendedMovies.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from "@tanstack/react-query"
+import { $api } from "shared/api/api";
+import { MOVIES_BASE_URL } from "shared/constants/api";
+import { GetTrendedMovieDTO } from "../api/types";
+import { mapTrendedMovieDTOToTrendedMovie } from "../mappers/movieMappers";
+import { HookReturnData } from "shared/types/hook";
+import { TrendedMovie } from "../model/types";
+
+export const useGetTrendedMovies = <ReturnData = TrendedMovie[] | null>(): HookReturnData<ReturnData> => {
+    const { data, isFetching, isError, isSuccess } = useQuery({
+        queryKey: ['trending'],
+        queryFn: async () => {
+            const response = await $api.get<GetTrendedMovieDTO[]>(`${MOVIES_BASE_URL}/movies/trended`);
```

Comment:
As I described at prev comment you'll move this logic to api method + I think you can skip MOVIES_BASE_URL here, because it's already set up on the base level